### DISCUSSION TITLE
METplus use cases and wrappers

### DISCUSSION THREAD

**Discussion by cbridge234** [dtcenter/METplus#1493] [Use Cases] [answered]
Hi METplus team,

Hope you are well. I wanted to ask what I think may be a basic question about METplus:
What is the difference between the METplus wrappers and METplus use cases?

I understand that the METplus wrappers provide a way to wrap individual MET tools in Python.
Is it then the case that those wrappers can be chained together to create workflows that we call use cases?

Any help much appreciated.
Thanks,
Chris

**Comment by jprestop** [comment, marked as answer]
Hi Chris (@cbridge234).

Thank you for your question.  The METplus Use Cases provide a low-level workflow which includes setting paths to data, dates to process, the order of processing, and configuration options.  The METplus Wrappers pass data and configuration options to the MET tools.  

I hope that helps to clarify.   Please let us know if you have any other questions.

Julie

**Reply by cbridge234** [reply]
Thanks Julie that does help clarify. Going through the [recent recorded training sessions](https://dtcenter.org/events/2021/metplus-training-series/agenda-recordings) has also helped.
Chris

**Reply by jprestop** [reply]
Great!  Thanks.  At this point I'm going to lock this conversation to prevent future posts. Our team has decided that once discussions have been answered, and the answer has been confirmed, we'll lock them. We want to encourage users to ask new questions in new discussions rather than posting to old ones. Hopefully that'll make the questions/answers easier for other users to follow.
So if/when more issues/questions arise, please feel free to start a new discussion.
